packages/plugin-animation/src/index.spec.ts

```diff
@@ -1,27 +1,27 @@
-import { jest } from "@jest/globals";
-import jsPsych from "jspsych";
+import { startTimeline } from "jspsych/tests/utils";
 
 import animation from ".";
 
 jest.useFakeTimers();
 
-describe("animation plugin", function () {
-  // SKIP FOR NOW
-  test.skip("displays stimuli", function () {
-    var animation_sequence = ["img/face_1.jpg", "img/face_2.jpg"];
+describe("animation plugin", () => {
+  test("displays stimuli", async () => {
+    const { getHTML, expectFinished } = await startTimeline([
+      {
+        type: animation,
+        stimuli: ["img/face_1.jpg", "img/face_2.jpg"],
+        render_on_canvas: false,
+      },
+    ]);
 
-    var trial = {
-      type: animation,
-      stimuli: animation_sequence,
-      render_on_canvas: false,
-    };
+    // TODO (bjoluc) Is it intended to wait before showing the first frame?
+    jest.advanceTimersByTime(250);
```

Comment:
@becky-gilbert @jodeleeuw ?

Reply:
I think this is a bug. #1885

Reply:
Thanks, I missed that! Leaving it for now then.